dmpr/path.py

```diff
@@ -0,0 +1,97 @@
+from .exceptions import InternalException
+
+
+def dict_reverse_lookup(d: dict, value):
+    return list(d.keys())[list(d.values()).index(value)]
+
+
+class Path:
+    """Auxiliary class to handle paths, appending and their link
```

Comment:
I prefer:

- new style classes (`class Foo(object)`)
- little different comment formatting

```
class Path(object):
    """
    Lorum ipsum shizle dizzle
    """

    def __init__(self):
        """
        Foo Bar Comment
        """
        pass
```

Reply:
In python 3 all classes are new style classes, but I can change that to the explicit form.